### PR TITLE
Possible grammar fixes (unsure)

### PR DESCRIPTION
**Pull request by giact**
These sentences sounded grammatically wrong to my ears when reading them, but I am quite unsure since English is not my first language.

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -100,7 +100,7 @@ \section{The Blockchain Paradigm} \label{ch:overview}
...
 
-Transactions are collated into blocks; blocks are chained together using a cryptographic hash as a means of reference. Blocks function as a journal, recording a series of transactions together with the previous block and an identifier for the final state (though do not store the final state itself---that would be far too big). They also punctuate the transaction series with incentives for nodes to \textit{mine}. This incentivisation takes places as a state-transition function, adding value to a nominated account.
+Transactions are collated into blocks; blocks are chained together using a cryptographic hash as a means of reference. Blocks function as a journal, recording a series of transactions together with the previous block and an identifier for the final state (though do not store the final state itself---that would be far too big). They also punctuate the transaction series with incentives for nodes to \textit{mine}. This incentivisation takes place as a state-transition function, adding value to a nominated account.
```
Comment (giact) [inline Paper.tex]: I simply changed *"takes places"* into *"takes place"*

File: Paper.tex [hunk 2]
```diff
@@ -137,7 +137,7 @@ \subsection{Value}
 
 \subsection{Which History?}
 
-Since the system is decentralised and all parties have an opportunity to create a new block on some older pre-existing block, the resultant structure is necessarily a tree of blocks. In order to form a consensus as to which path, from root (the genesis block) to leaf (the block containing the most recent transactions) through this tree structure, known as the blockchain, there must be an agreed-upon scheme. If there is ever a disagreement between nodes as to which root-to-leaf path down the block tree is the `best' blockchain, then a \textit{fork} occurs.
```
Comment (giact) [inline Paper.tex]: That sentence in the middle boils down to: _"In order to form a consensus as to which path [...] there must be an agreed-upon scheme"_

I feel like there should be a verb in there, tied to _"path"_. I opted for _"is to be considered canonical"_, but then I had to move things around a little.

File: Paper.tex (hunk 1 above)
Comment (gavofyork) [inline Paper.tex]: correct - please supply as individual PR.

Comment (giact) [inline Paper.tex]: Done: #176

File: Paper.tex (hunk 2 above)
Comment (gavofyork) [inline Paper.tex]: still doesn't read particularly well since "canonical" is too far apart from "blockchain".

**Comment by axic** [comment]
@giact can you please rebase and apply the requested changes?

**Comment by giact** [comment]
Sorry for the late reply.

I am confused on what changes I am expected to apply (but please be patient with me cause I have not been following the yellow paper at all since last summer).

I thought the good one (0a4370a9ff18cee122c2361e43c44e09d52d92f3) got already merged in #176, while 1af91609775a08e6e6d60fa3b682d229d33322f2 was to be discarded cause, as @gavofyork said, it doesn't read well.

**Comment by nicksavers** [comment]
As said, the first part is already merged in a different PR
The other sentence does not flow well, but this PR also needs a rebase.
Better to create a new PR for https://github.com/ethereum/yellowpaper/commit/1af91609775a08e6e6d60fa3b682d229d33322f2.

**Pull request closed without merging**